test_runner/fixtures/neon_fixtures.py

```diff
@@ -2921,6 +2923,8 @@ def http_query(self, query, args, **kwargs):
         password = quote(kwargs["password"])
         expected_code = kwargs.get("expected_code")
 
+        log.info(f"Executing http query: {query}")
+
```

Comment:
We might consider removing those...

Reply:
It's a log during a test. No need to remove